### ISSUE TITLE
Rule errors reported with ESLint 2.1.0

### ISSUE THREAD

**Issue by marcinant** [ntwb/eslint-config-wordpress#1]
I've applied this config and now I got some errors such as:

```
Rule 'no-empty-label' was removed and replaced by: no-labels
Rule 'space-after-keywords' was removed and replaced by: keyword-spacing
Rule 'space-return-throw-case' was removed and replaced by: keyword-spacing
```

Could you please update some settings for ES Lint 2.x ?

**Comment by ntwb** [comment]
Thanks, I'll try to get to this before the weekend :+1:

**Comment by jahvi** [comment]
I submitted a PR #2, looks good on my end but it might need some testing.

**Comment by ntwb** [comment]
I've merged #2, I'm just adding some tests for Travis CI, hope to get that sorted in the next couple of days  to make a new release, sorry for the delay.

**Comment by adamsilverstein** [comment]
@ntwb  Also seeing these errors and fixed locally, I don't think these changes are in npm yet? Thanks for the package!

**Comment by ntwb** [comment]
Apologies everyone, I've been working on `eslint-plugin-wordpress` with multiple _shared configs_ and had hoped an upstream ESLint issue would have been resolved by now that has not, as such sharing configs between "shared plugins and shared configs" is not easily achievable at this time.

So to get both `eslint-config-wordpress` and `eslint-plugin-wordpress` repos up to date and able to be iterated on I've created a roadmap that I think will get things back on track.

v1, v2, and v3 Roadmap https://github.com/ntwb/eslint-config-wordpress/issues/8

**Comment by ntwb** [comment]
I've just released v1.0.0:
• https://www.npmjs.com/package/eslint-config-wordpress
• https://github.com/ntwb/eslint-config-wordpress/releases

For any issues please check the roadmap notes in #8 and if needed create a new issue

Going to mark this issue now resolved, and thanks everyone for your patience.
